src/utils/components/PolicyForm/PolicyInterface.tsx

```diff
@@ -195,17 +199,42 @@ const PolicyInterface: FC<PolicyInterfaceProps> = ({
         <Checkbox
           label={t('IPV4')}
           id={`policy-interface-ip-${id}`}
-          isChecked={!!policyInterface.ipv4}
+          isChecked={policyInterface.ipv4?.enabled}
```

Comment:
Is `ipv4` guaranteed to be there? If not, could you please add optional chaining there and the other places it's used?